### DISCUSSION TITLE
Simple Vector

### DISCUSSION THREAD

**Discussion by JustKohar** [mrdbourke/pytorch-deep-learning#525] [Q&A]
Hello, I am new to coding and am following a video learning how to use Pytorch so this may be a stupid question. Does anyone know why this isn't working? I tried importing the torch class also and it threw the same error. 

`## Vector
import torch
vector = torch.tensor[(7, 7)]
vector`

It's throwing this error.

TypeError                                 Traceback (most recent call last)
[<ipython-input-16-2b124a62f8e1>](https://localhost:8080/#) in <cell line: 3>()
      1 ## Vector
      2 import torch
----> 3 vector = torch.tensor[7, 7];
      4 vector

TypeError: 'builtin_function_or_method' object is not subscriptable

Thanks for the help!

**Comment by Perian-Yan** [comment]
Hi, I think you forget the parentheses and there's no need for a semicolon at the end.
``` python
import torch
vector = torch.tensor([7,7])
vector
```
